Gruntfile.js

```diff
@@ -0,0 +1,206 @@
+module.exports = function (grunt) {
+	'use strict';
+
+	/**
+	* LOAD TASKS
+	*/
+
+	require( 'matchdep' ).filterDev( 'grunt-*' ).forEach( grunt.loadNpmTasks );
+	
+	/**
+	* CONFIGURATION
+	*/
+	grunt.initConfig({
+		pkg: grunt.file.readJSON('package.json'),
+		jshint: {
+			dist: [
+				'js/*.js'
+			],
+			options: {
+				"-W041": false,
+				curly: true,
+				eqeqeq: false,
+				immed: true,
+				latedef: false,
+				newcap: true,
+				noarg: true,
+				sub: true,
+				undef: false,
+				boss: true,
+				eqnull: false,
+				browser: true,
+				jquery: true,
+				globals: {
+					require: true,
+					define: true,
+					requirejs: true,
+					describe: true,
+					expect: true,
+					it: true
+				}
+			}
+		},
+		clean: {
+			options: {
+				force: true
+			},
+			dist: [
+				'css/dist/cloud-search.min.css',
+				'css/dist/cloud-search-suggest.min.css',
+				'css/dist/images/',
+				'js/dist/cloud-search.min.js',
+				'js/dist/cloud-search-admin-help.min.js',
+				'js/dist/cloud-search-admin-manage.min.js',
+				'js/dist/cloud-search-admin-settings.min.js',
+				'js/dist/cloud-search-locale-en-US.min.js',
+				'js/dist/cloud-search-locale-it-IT.min.js',
+				'js/dist/cloud-search-suggest.min.js',
+			],
+			deploy: [
+				'deploy'
+			]
+		},
+		copy: {
+			dist: {
+				files: [
+					{
+						expand: true,
+						cwd: 'css/images/',
+						src: ['**'],
+						dest: 'css/dist/images/'
+					}
+				]
+			},
+			deploy: {
+				files: [{
+					expand: true,
+					src: [
+						'**',
+						'!**/assets/**',
+						'!**/bin/**',
+						'!**/deploy/**',
+						'!**/sass/**',
+						'!**/node_modules/**',
+						'!**/tests/**',
+						'!config.rb',
+						'!Gruntfile.js',
+						'!package.json',
+						'!phpunit.xml',
+						'!README.md'
+					],
+					dest: 'deploy/'
+				}],
+			}
+		},
+		cssmin: {
+			dist: {
+				files: [
+					{
+						src: 'css/cloud-search.css',
+						dest: 'css/dist/cloud-search.min.css'
+					},
+					{
+						src: 'css/cloud-search-suggest.css',
+						dest: 'css/dist/cloud-search-suggest.min.css'
+					}
+				]
+			}
+		},
+		wp_readme_to_markdown: {
```

Comment:
This is the settings that allow the WP readme.txt to be converted into a nice GitHub style README.md. Not really necessary but it makes the repository look pretty on GitHub.

Reply:
Yes, it's a good idea... it's a GitHub repository..